### PR TITLE
minor proofreading, prepare for backport

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: doc/source/operations.rst [hunk 1]
```diff
@@ -237,19 +237,20 @@ following information to the "osh" section under "charts":
 Reboot Compute Host
 ===================
 
-Before reboot compute host, shutdown all Nova VM(s) from that compute host.
+Before rebooting the compute host, shutdown all Nova VMs from that compute
```
Comment (jgu17) [inline doc/source/operations.rst]: shutdown should be shut down?

Comment (csymons-suse) [reply doc/source/operations.rst]: yes. It should be shut down. fixed.

Comment (jgu17) [reply doc/source/operations.rst]: don't see the fix?

**Comment by csymons-suse** [comment]
no longer applicable.

**Pull request closed without merging**